indico/modules/events/contributions/templates/display/user_contribution_list.html

```diff
@@ -5,41 +5,37 @@
-                <th>{% trans %}Title{% endtrans %}</th>
-                <th></th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for contrib in contributions -%}
-                <tr>
-                    <td>
-                        <a class="js-mathjax" href="{{ url_for('.display_contribution', contrib) }}">{{ contrib.title }}</a>
-                    </td>
-                    <td>
-                        <div class="group right">
-                            {% if contrib.can_manage(session.user) -%}
-                                <a href="#" class="icon-edit js-edit-contribution"
-                                   title="{% trans %}Edit this contribution{% endtrans %}"
-                                   data-title="{% trans title=contrib.title %}Edit contribution '{{ title }}'{% endtrans %}"
-                                   data-href="{{ url_for('.manage_update_contrib', contrib) }}"
-                                   data-ajax-dialog
-                                   data-reload-after></a>
-                            {%- endif %}
-                        </div>
-                    </td>
-                </tr>
-            {%- else -%}
-                <tr>
-                    <td colspan="2">
-                        {%- trans %}You don't have any contributions at the moment.{% endtrans -%}
-                    </td>
-                </tr>
-            {%- endfor %}
-        </tbody>
-    </table>
+    <section>
+        <div class="i-box no-padding">
+            <div class="i-box-content">
+                <ul class="group-list no-content-before with-buttons">
+                    {% for contrib in contributions %}
+                        <li class="flexrow f-j-space-between">
+                            <span class="text">
+                                <a class="js-mathjax" href="{{ url_for('.display_contribution', contrib) }}">
+                                    {{- contrib.title -}}
+                                </a>
+                            </span>
+                            <div class="group">
+                                {% if contrib.can_manage(session.user) -%}
+                                    <a href="#" class="icon-edit js-edit-contribution"
+                                       title="{% trans %}Edit this contribution{% endtrans %}"
+                                       data-title="{% trans title=contrib.title %}Edit contribution '{{ title }}'{% endtrans %}"
+                                       data-href="{{ url_for('.manage_update_contrib', contrib) }}"
+                                       data-ajax-dialog
+                                       data-reload-after></a>
+                                {%- endif %}
+                            </div>
+                        </li>
+                    {% else %}
+                        <li>
+                            {%- trans %}You don't have any contributions at the moment.{% endtrans -%}
```

Comment:
This could nicely fit in one like (like you did in `contribution_display.html` for the `<h3>`), but that's just some personal preferences, feel free to use your own style! But then be consistent :stuck_out_tongue_winking_eye: